src/components/avatar.tsx

```diff
@@ -12,7 +12,7 @@ export function Avatar(props: AvatarProps) {
     <img
       src={imageUrl}
       alt="userPhoto"
-      className={twJoin("avatar", className)}
+      className={twJoin("rounded-full border-white border-2", className)}
```

Comment:
Could you please make the implementation of the classes outside of the className property?

Reply:
we did that on https://github.com/cristianoprogramador/FE_SH/pull/12